packages/vuetify/src/components/VExpansionPanel/VExpansionPanels.tsx

```diff
@@ -0,0 +1,51 @@
+// Styles
+import './VExpansionPanel.sass'
+
+// Composables
+import { makeTagProps } from '@/composables/tag'
+import { makeGroupProps, useGroup } from '@/composables/group'
+import { makeThemeProps, useTheme } from '@/composables/theme'
+
+// Utilities
+import { computed } from 'vue'
+import { defineComponent } from '@/util'
+
+// Types
+import type { InjectionKey } from 'vue'
+import type { GroupItemProvide } from '@/composables/group'
+
+export const VExpansionPanelSymbol: InjectionKey<GroupItemProvide> = Symbol.for('vuetify:v-expansion-panel')
+
+export default defineComponent({
+  name: 'VExpansionPanels',
+
+  props: {
+    variant: {
+      type: String,
+      default: 'default',
+      validator: (v: any) => ['default', 'accordion', 'inset', 'popout'].includes(v),
```

Comment:
should we move this to a variable such as, **const allowedVariants = ['default', 'accordion', 'inset', 'popout'] as const**

Reply:
So far it's only every used here

Reply:
Other components do `String as PropType<Variant>` so it can be fully type safe